src/main/java/de/doubleslash/keeptime/view/ManageWorkController.java

```diff
@@ -239,6 +240,16 @@ public void changed(final ObservableValue<? extends String> observable, final St
 
       });
 
+      // on
+      projectComboBox.setOnKeyReleased((ke) -> {
+         if (ke.getCode() == KeyCode.ENTER && projectComboBox.getSelectionModel().isEmpty()) {
+            if (!projectComboBox.getItems().isEmpty()) {
+               projectComboBox.getSelectionModel().selectFirst();
```

Comment:
this makes the UI respond weird. its looks like an endless call loop or something, the cursor starts flickering if I select a project with "Enter"